### PR TITLE
(PA-5999) modify configs for sles11-x86_64

### PR DESCRIPTION
**Pull request by amitkarsale**
modified configs for sles11-x86_64. Please review for  `plat.add_build_repository` too.

### CODE REVIEW

File: configs/platforms/sles-11-x86_64.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 platform "sles-11-x86_64" do |plat|
-  plat.servicedir "/etc/init.d"
+  plat.servicedir "/usr/lib/systemd/system"
```
Comment (joshcooper) [inline configs/platforms/sles-11-x86_64.rb:2]: I believe SLES 11 still uses the older init style, not systemd

```
tomblike-wait:~ # cat /etc/os-release 
NAME="SLES"
VERSION="11.4"
VERSION_ID="11.4"
PRETTY_NAME="SUSE Linux Enterprise Server 11 SP4"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:11:4"
tomblike-wait:~ # ls -d /etc/init.d/rc*
/etc/init.d/rc     /etc/init.d/rc1.d  /etc/init.d/rc3.d  /etc/init.d/rc5.d  /etc/init.d/rcS.d
/etc/init.d/rc0.d  /etc/init.d/rc2.d  /etc/init.d/rc4.d  /etc/init.d/rc6.d
tomblike-wait:~ # ls /usr/lib/systemd
ls: cannot access /usr/lib/systemd: No such file or directory
```

File: configs/platforms/sles-11-x86_64.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   plat.defaultdir "/etc/sysconfig"
-  plat.servicetype "sysv"
-  plat.add_build_repository "http://osmirror.delivery.puppetlabs.net/sles-11-deps-x86_64/sles-11-deps-x86_64.repo"
```
Comment (joshcooper) [inline configs/platforms/sles-11-x86_64.rb:5]: I think we want to keep this so that we install the packages below, like `gcc`, from our internal mirror.

**Comment by joshcooper** [comment]
You can drop the `java-1_7_1-ibm-devel` package, see abeb69129f6105a5355924e01df36405afa0cce5

**Comment by joshcooper** [comment]
@amitkarsale This looks to have been superseded by #787, can we close this one?

**Comment by shubhamshinde360** [comment]
Resolved under https://github.com/puppetlabs/puppet-runtime/pull/788

**Pull request closed without merging**